Fix/et 1058 fix skipped tests (#2865)
the-events-calendar/event-tickets: readme.txt

== Changelog ==

= TBD

> * Tweak - Updated sharedCapacity prop type to number. This avoids warnings in jest testing [ET-1058]

Reviewer: Since the changelog is public, I don't think we need to mention anything about internal tests.
```suggestion
* Tweak - Updated sharedCapacity prop type to number within the tickets block to avoid warning logs. [ET-1058]
```

Author: Thanks @codingmusician. Suggestion applied
